src/wmnds/patterns/content-cards/travel-updates/_travel-updates.njk

```diff
@@ -0,0 +1,61 @@
+{% macro wmndsContentCardTravelUpdates(params) %}
+{# imports #}
+{% from "wmnds/components/content-card/_content-card.njk" import wmndsContentCard %}
+{% from "wmnds/components/disruption-indicator/large/_large.njk" import wmndsDisruptionIndicatorLarge as largeDisruption %}
+{% from "wmnds/components/disruption-indicator/medium/_medium.njk" import wmndsDisruptionIndicatorMedium as mediumDisruption %}
+{% from "wmnds/components/message/help/_help.njk" import wmndsMsgHelp %}
+{# Set #}
+{% set largeDisruptionsAreas = ["Hagley Road", "Ring Road"] %}
+{% set buses = ["9", "10", "12", "12A", "13", "13A", "126", "X8", "X10", "23"] %}
+
+{% call wmndsContentCard({ variant: "wmnds-content-card--travel-updates"}) %}
+    <!-- .wmnds-col-* is just used to set the width in the example -->
+    <div class="wmnds-p-sm wmnds-col-10-24">
```

Comment:
```suggestion
    <div class="wmnds-p-sm wmnds-col-1 wmnds-col-md-10-24">
```

Test this class change, as it breaks on mobile currently. And all components/patterns should be flexible enough that you can use them at multiple widths. So it may be a case of changing to wmnds-col-1 and making use of more space.